extensions/typescript-language-features/web/webServer.ts

```diff
@@ -248,24 +261,50 @@ function createServerHost(extensionUri: URI, logger: ts.server.Logger, apiClient
 				return request.status === 200;
 			}
 
+			const uri = toResource(path);
+			let ret = null;
 			try {
-				return fs.stat(toResource(path)).type === FileType.File;
-			} catch (error) {
-				logNormal('Error fs.fileExists', { path, error: error + '' });
-				return false;
+				ret = fs.stat(uri).type === FileType.File;
+			} catch (_error) {
+				try {
+					console.log('fs.fileExists fallback', { path, mapped: mapUri(uri, 'vscode-node-modules') });
+					ret = fs.stat(mapUri(uri, 'vscode-node-modules')).type === FileType.File;
+				} catch (_error) {
+				}
 			}
+			return ret || false;
```

Comment:
Is this `|| false` doing something?

Reply:
Kinda, but it was better to write it a different way, so I've changed it. :)